src/test/java/cloud/util/TaskListTest.java

```diff
@@ -0,0 +1,55 @@
+package cloud.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import cloud.task.Task;
+
+public class TaskListTest {
+    private TaskList taskList;
+    private TaskStub task;
+
+    @BeforeEach
+    void setUp() {
+        this.taskList = new TaskList();
+        this.task = new TaskStub("Test task");
+    }
+
+    @Test
+    public void add_addTask_increaseTaskCount() {
```

Comment:
Would it be better to rename it as "addTaskTest" according to the JUnit naming standard given?